rust-src/id/src/account_holder.rs

```diff
@@ -323,6 +326,373 @@ pub fn generate_pio<P: Pairing, C: Curve<Scalar = P::ScalarField>>(
     ))
 }
 
+pub fn generate_pio<P: Pairing, C: Curve<Scalar = P::ScalarField>>(
+    // TODO: consider renaming this function
```

Comment:
Maybe now is the time? :) Otherwise we should give up on it and delete the comment.